app.js

```diff
@@ -72,7 +72,7 @@ function processLine (line, prefix, defaultTags) {
     if (process.env.DEBUG) {
       console.log('Processing router metrics');
     }
-    let tags = tagsToArr(_.pick(line, ['dyno', 'method', 'status', 'path', 'host', 'code', 'desc', 'at']));
+    let tags = tagsToArr(_.pick(line, ['dyno', 'method', 'status', 'host', 'code', 'desc', 'at']));
```

Comment:
Some, but there are bunch of bloats coming from paths e.g.

```
/admin/addresses/{id}
/admin/brns/{id}
/admin/customers/{id}
...
```
I can imagine carefully collecting and capturing patterns in all of web's urls to make `path` tag more useful, but that'd be a pain to maintain + if we miss any url that causes bloat in custom metrics, a large datadog bill awaits us.